src/components/Collection/SeriesTopPanel.tsx

```diff
@@ -25,24 +26,24 @@ const SeriesTag = React.memo(({ text, type }: { text: string, type: 'User' | 'An
   const dispatch = useDispatch();
   const navigate = useNavigate();
   const handleClick = useEventCallback(() => {
-    dispatch(resetFilter());
```

Comment:
Why is this removed?
This will cause the tag condition to be added to the existing filter

Reply:
I expect some frustration if people have like 5 filters set up and it removes them all but it is linking from another page in this case so I'm not sure what is better (I put it back though)